WebClient/src/components/moduleControl/moduleControlToggle/moduleControlToggleView.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * ModuleControlToggleView. Represents a toggle button that shows or hides the ModuleControlWindowView.
+ */
+
+/* globals L */ 
+
+import './moduleControlToggle.css';
+import moduleControlToggleHtml from './moduleControlToggle.html';
+import View from '../../../core/view';
+
+/* globals L, d3 */
+
+var ControlView = L.Control.extend(View.prototype);
+
+var ModuleControlToggleView = ControlView.extend({    
+    options: {
+        collapsed: true,
+        position: 'bottomright',
+        autoZIndex: true
+    },
+
+    onInitialize: function (opts) {
+        if (!opts) throw new Error('opts is not provided');
+        if (!opts.toggleViewModel) throw new Error('opts.toggleViewModel is nleaflet-control-model-toggleot provided');
```

Comment:
The error message here does not feel correct. Is that so?

Reply:
:)
👍